packages/oas-utils/src/entities/spec/request-examples.ts

```diff
@@ -287,7 +287,12 @@ export function convertExampleToXScalar(example: RequestExample) {
 export function createParamInstance(param: RequestParameter) {
   const schema = param.schema as any
   const keys = Object.keys(param?.examples ?? {})
-  const firstExample = keys.length ? param.examples?.[keys[0]!] : null
+  const firstExample =
```

Comment:
This is getting a bit crazy, I almost think we should turn this rule on

https://biomejs.dev/linter/rules/no-nested-ternary/#_top